src/t_string.c

```diff
@@ -299,7 +299,7 @@ int parseExtendedStringArgumentsOrReply(client *c, int *flags, int *unit, robj *
             addReplyErrorObject(c,shared.syntaxerr);
             return C_ERR;
         }
-        /* clang-format off */
+        /* clang-format on */
```

Comment:
No, if we delete  this one, it will significantly change the "if-else" chain starting at line 229. Hence this is one of two things I decided to keep unformatted.